model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -189,7 +189,9 @@ def groupconv_to_conv(op: Node):
         assert weights_node.in_port(0).get_source().data.get_shape() == new_shape, \
             'Weight shape and calculated shape mismatch in GroupConv node {}.'.format(op.name)
         op.in_port(1).disconnect()
-        weights_node.in_port(0).get_source().get_connection().set_destination(op.in_port(1))
+        # We don't use set_destination method here to support case with multiple destinations of source port
```

Comment:
I think this comment is about your fix. That says about what we had in the past and is not relevant.

Reply:
Comment updated.